src/components/ItaliaTheme/View/VenueView.jsx

```diff
@@ -311,6 +311,15 @@ const VenueView = ({ content }) => {
                 title={intl.formatMessage(messages.ulteriori_informazioni)}
               />
             )}
+
+             {/* SEDE DI */}
+             {content.sede_di?.length > 0 && (
```

Comment:
Non hai il check su content truthy, quindi:
`content?.sede_di`
Ho dato un'occhio rapido al file intero e ce ne sono altri di mancanti, anche nello use effect, controlla :)

Reply:
Avevo già controllato la casistica, se sedi_di è null content.sede_di?.length è null e quindi content.sede_di?.length > 0 è false e quindi non entra nell'if.

Reply:
ok, se content per qualche motivo e' null o undefined non si rompe? Potrebbe dirti "non esiste la prop 'sedi_di' per null o undefined"

Reply:
si in quel caso si spaccca, correggi se sbaglio ma se content è nullo vuol dire che non si è riusciti a prendere il fullobject dell'oggetto e questo sarebbe un bel problema, forse è bene che si spacchi la pagina se c'è un problema del genere.
Sto provando a vedere le altre view non mi sembra che nessuna controlli content

Reply:
In realtà in qualche caso è controllato ma non molte volte, mi sembra più una cosa fatta sporadicamente